src/GlobalSettings.gd

```diff
@@ -73,6 +73,7 @@ func _enter_tree() -> void:
 	load_settings()
 	load_user_data()
 	DisplayServer.window_set_mode(save_data.window_mode)
+	get_window().wrap_controls = true
```

Comment:
This enables automatic window minimum size.
I didnt know where else to put this

Reply:
What does this mean?

Reply:
you cant resize the window to something too small to fit the content anymore

Reply:
This is cool, let's keep it, just add a comment.

Reply:
its only going to be useful with the scaling mode change tho.
ill add a comment right now

Reply:
Just tested it, no, it's useful without the scaling mode change. Also, I still want to change the scaling mode, I just want to not do it without implementing customizable theme scale.